share/docker/casa-run/ubuntu-18.04/build_sip_pyqt.sh

```diff
@@ -0,0 +1,135 @@
+#! /bin/bash
+
+set -e
+set -x
+
+SIP_VERSION=4.19.15
+PYQT_VERSION=5.12.1
+PRE_4_19=
+PRE_4_18=
+PREFIX=/usr/local/sip-$SIP_VERSION
+BUILD=/tmp/sipbuild
+PATCH_FILE=ftp://ftp.cea.fr/pub/dsv/anatomist/3rdparty/1.0.0/sources/pyqt_patch
+
+PY3=3.6m
+PY3_S=3.6
+QMAKE=/usr/lib/qt5/bin/qmake
+
+mkdir "$BUILD"
+cd "$BUILD"
+if [ -z "$PRE_4_19" ]; then
```

Comment:
Do we still need to support sip versions older than 4.19 ?

Reply:
Probably not, although only @denisri knows for sure. I removed support for SIP < 4.19 in https://github.com/brainvisa/casa-distro/commit/5de207e08da34b77a82395676ae9ee68c6d40024, we can revert this change if needed.